tensorflow/tools/dockerfiles/dockerfiles/devel-cpu.Dockerfile

```diff
@@ -23,8 +23,11 @@ ARG UBUNTU_VERSION=18.04
 
 FROM ubuntu:${UBUNTU_VERSION} AS base
 
+ARG DEBIAN_FRONTEND="noninteractive"
+
 RUN apt-get update && apt-get install -y --no-install-recommends \
         build-essential \
+        clang-format \
```

Comment:
We could install `clang-format-10` if we prefer `dry-run` but we always need to sync version with the CI Sanity VM